packages/admin/admin/src/table/filterbar/filterBarActiveFilterBadge/FilterBarActiveFilterBadge.tsx

```diff
@@ -1,35 +1,53 @@
-import { ComponentsOverrides, Theme, Typography } from "@mui/material";
-import { WithStyles, withStyles } from "@mui/styles";
+import { ComponentsOverrides, Typography } from "@mui/material";
+import { css, styled, Theme, useThemeProps } from "@mui/material/styles";
+import { ThemedComponentBaseProps } from "helpers/ThemedComponentBaseProps";
 import * as React from "react";
 
-import { FilterBarActiveFilterBadgeClassKey, styles } from "./FilterBarActiveFilterBadge.styles";
+export type FilterBarActiveFilterBadgeClassKey = "hasValueCount";
 
-/**
- * @deprecated Use MUI X Data Grid in combination with `useDataGridRemote` instead.
- */
-export interface FilterBarActiveFilterBadgeProps {
+const HasValueCount = styled("div", {
+    name: "CometAdminFilterBarActiveFilterBadge",
+    slot: "hasValueCount",
```

Comment:
@jamesricky shouldn't this be `root`?

Reply:
No, for compatibility this needs to stay `hasValueCount` as was the className in the previous styling method. 
Changing it would break the `styleOverrides` in the theme, if used in an app.